jdbc/src/main/java/software/amazon/timestream/jdbc/TimestreamColumnsResultSet.java

```diff
@@ -64,6 +64,9 @@ public class TimestreamColumnsResultSet extends TimestreamBaseResultSet {
     TimestreamDataType.createColumnInfo(TimestreamDataType.VARCHAR, "IS_AUTOINCREMENT"),
     TimestreamDataType.createColumnInfo(TimestreamDataType.VARCHAR, "IS_GENERATEDCOLUMN"));
 
+  private final int TABLE_SCHEM_INDX = 2;
```

Comment:
nit. comment with the doc, explaining this constant (and constant below)

Reply:
Updated the comment